.github/workflows/ci.yaml

```diff
@@ -44,3 +44,39 @@ jobs:
               uses: "docker://oskarstark/phpstan-ga"
               with:
                   args: analyse
+
+    auto-merge:
+        name: "Auto-Merge"
+
+        runs-on: "ubuntu-latest"
+
+        needs:
+            - "php-cs-fixer"
+            - "composer-normalize"
+            - "phpstan"
+
+        if: >
+            github.event_name == 'pull_request' &&
+            github.event.pull_request.draft == false && (
+              github.event.action == 'opened' ||
+              github.event.action == 'reopened' ||
+              github.event.action == 'synchronize'
+            ) && (
+              github.actor == 'dependabot[bot]'
+            )
+
+        steps:
+            - name: "Merge pull request"
+              uses: "actions/github-script@v2"
+              with:
+                  github-token: "$"
```

Comment:
What is this?

Reply:
IIRC correct it uses a/the token of the repository. Is that right @localheinz?

Reply:
Ah no it is a placeholder in template, wait